build.sbt

```diff
@@ -952,10 +952,15 @@ lazy val `json-rpc-server` = project
     libraryDependencies ++= akka ++ logbackTest,
     libraryDependencies ++= circe,
     libraryDependencies ++= Seq(
-      "io.circe"                   %% "circe-literal" % circeVersion,
-      "com.typesafe.scala-logging" %% "scala-logging" % scalaLoggingVersion,
+      "io.circe"                   %% "circe-literal"   % circeVersion,
+      "com.typesafe.scala-logging" %% "scala-logging"   % scalaLoggingVersion,
       akkaTestkit                   % Test,
-      "org.scalatest"              %% "scalatest"     % scalatestVersion % Test
+      "org.scalatest"              %% "scalatest"       % scalatestVersion % Test,
+      "junit"                       % "junit"           % junitVersion     % Test,
+      "com.github.sbt"              % "junit-interface" % junitIfVersion   % Test,
+      "org.apache.httpcomponents"   % "httpclient"      % "4.4.1"          % Test,
+      "org.apache.httpcomponents"   % "httpcore"        % "4.4.16"         % Test,
+      "commons-io"                  % "commons-io"      % "2.14.0"         % Test
```

Comment:
We have `commonsIoVersion` variable to use the same library version across all the projects. Please create appropriate variables for `httpcomponents` it will simplify the library upgrade process in the future.